example/ndpiReader.c

```diff
@@ -2472,6 +2472,8 @@ static void setupDetection(u_int16_t thread_id, pcap_t * pcap_handle) {
 			  NDPI_LRUCACHE_BITTORRENT, 32768);
   /* Enable/disable LRU caches TTL here */
 
+  /* Set aggressiviness here */
```

Comment:
I would suggest to provide a command line option instead.

Reply:
Done. Added a generic `-Z proto_name:aggressiveness_value` to `ndpiReader`